Fireworks/Calo/plugins/FWFTLRecHitProxyBuilder.cc

```diff
@@ -0,0 +1,60 @@
+#include "Fireworks/Calo/plugins/FWCaloRecHitDigitSetProxyBuilder.h"
+#include "DataFormats/FTLRecHit/interface/FTLRecHitCollections.h"
+
+class FWFTLRecHitProxyBuilder : public FWCaloRecHitDigitSetProxyBuilder
+{
+public:
+   FWFTLRecHitProxyBuilder( void ) { invertBox(true); }
+   virtual ~FWFTLRecHitProxyBuilder( void ) {}
+
+   REGISTER_PROXYBUILDER_METHODS();
+
+private:
+   FWFTLRecHitProxyBuilder( const FWFTLRecHitProxyBuilder& );
+   const FWFTLRecHitProxyBuilder& operator=( const FWFTLRecHitProxyBuilder& );
+};
+
+REGISTER_FWPROXYBUILDER( FWFTLRecHitProxyBuilder, FTLRecHitCollection, "FTL RecHit", FWViewType::kISpyBit );
+
+
+// AMT: Refelct box. Previously used energyScaledBox3DCorners()
+
+/*
+void
+FWHBHERecHitProxyBuilder::build( const FWEventItem* iItem, TEveElementList* product, const FWViewContext* vc)
+{
+   m_plotEt = vc->getEnergyScale()->getPlotEt();
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant